buildkite/aggregate_incompatible_flags_test_result.py

```diff
@@ -31,6 +31,23 @@
 
 FLAG_LINE_PATTERN = re.compile(r"\s*(?P<flag>--\S+)\s*")
 
+MODULE_VERSION_PATTERN = re.compile(r'(?P<module_version>[a-z](?:[a-z0-9._-]*[a-z0-9])?@[^\s]+)')
+
+BAZEL_TEAM_OWNED_MODULES = [
```

Comment:
ok, I'm confused, these changes are also in the other PR

Reply:
Yeah, this was stacked on the previous one. Since `BAZEL_TEAM_OWNED_MODULES` is so far only used in `aggregate_incompatible_flags_test_result.py`, I prefer to keep it there for now. We can refactor this if we need it somewhere else.